pages/api/trees/rainfall.ts

```diff
@@ -23,22 +23,29 @@ export default async function handler(
             .status(400)
             .json({ error: 'Missing tree_id search parameter' })
         }
-        // TODO: Test rainfall function once it is implemented in the database
-        const { data: rainfalls, error } = await postgrest.rpc('rainfall', {
-          tree_id: searchParams.get('tree_id') as string,
-        })
-        if (error) {
-          throw new Error(error.message)
+
+        const { data: rainfall_mat, error: rainfall_mat_error } =
```

Comment:
How about we remove the `_mat` suffix? In the end we'll get back simply `rainfall` data, no matter if it's from a materialized view or a regular table.

Reply:
You are right. Was a leftover from testing function and view side by side